src/pages/Base.page.js

```diff
@@ -1,5 +1,5 @@
 export class BasePage {
-    url = '';
+    url = 'https://www.saucedemo.com/';
```

Comment:
Its better to move this to playwright.config -> use.baseUrl and leave relatibe url here ('').

Reply:
Oh, we have such setup in cofig, no need to set url here.